tools/get-certificate/main.go

```diff
@@ -41,6 +42,9 @@ func main() {
 
 	ctx, done := signalcontext.OnInterrupt()
 
+	if os.Getenv("LOG_LEVEL") == "" {
```

Comment:
It’d be better just to use the debugger logger instead of setting the env. Just don’t use logging.NewFromEnv and use the NewDebug method (I think there’s one)

Reply:
this still allows the user to specify